apps/public-docsite-v9/src/Concepts/Migration/FromV8/ComponentMapping.stories.mdx

```diff
@@ -11,70 +11,70 @@ Here is the mapping of v8 components to their v9 component replacement or equiva
+| ColorPicker            |                                                                                                                                              |
+| ComboBox               | [ComboBox](/docs/components-combobox--default)                                                                                               |
+| ContextualMenu         | [Menu](/docs/components-menu-menu--default) with Button as the MenuTrigger                                                                   |
+| DefaultButton          | [Button](/docs/components-button-button--default)                                                                                            |
+| DefaultButton (anchor) | [Button](/docs/components-button-button--default)                                                                                            |
+| DefaultButton (menu)   | [MenuButton](/docs/components-button-menubutton--default)                                                                                    |
+| DatePicker             | [DatePickerCompat](/docs/compat-components-datepicker--default)                                                                              |
+| DetailsList            | [~DataGrid](/docs/components-datagrid--default)                                                                                              |
+| Dialog                 | [Dialog](/docs/components-dialog--default)                                                                                                   |
+| DocumentCard           | [~Card](/docs/components-card-card--default)                                                                                                 |
+| Dropdown               | [~Dropdown](/docs/components-dropdown--default)                                                                                              |
+| Facepile               | [~AvatarGroup](/docs/components-avatargroup--default)                                                                                        |
+| GroupedList            | [~Tree](/docs/components-tree--default)                                                                                                      |
+| Icon                   | [(icons in react-icons)](/docs/concepts-developer-icons-icons--page)                                                                         |
+| IconButton             | [Button](/docs/components-button-button--default)                                                                                            |
+| Image                  | [Image](/docs/components-image--default)                                                                                                     |
+| Label                  | [Label](/docs/components-label--default)                                                                                                     |
```

Comment:
Keytips missing or maybe we don't want to mention that one either :)

Reply:
Who the heck made this list, so many things missing 👀?! /s